strutil/strutil.go

```diff
@@ -182,3 +183,18 @@ func CommaSeparatedList(str string) []string {
 	}
 	return filtered
 }
+
+// ElliptRight returns a string that is at most n runes long,
+// replacing the last rune with an ellipsis if necessary. If N is less
+// than 1 it's treated as a 1.
+func ElliptRight(str string, n int) string {
+	if n < 1 {
```

Comment:
maybe if n > 1 default to the last rune in that string? And return the original size